test/AutoDiff/refcounting.swift

```diff
@@ -71,7 +71,6 @@ _ = pullback(at: Vector.zero, in: side_effect_release_zero)
 // CHECK:  [[ZERO_GETTER:%.*]] = function_ref @$s11refcounting6VectorV4zeroACvgZ
 // CHECK:  [[ZERO:%.*]] = apply [[ZERO_GETTER]]({{%.*}}) : $@convention(method) (@thin Vector.Type) -> @owned Vector
 // CHECK:  store [[ZERO]] to [[BUF_ACCESS]] : $*Vector
-// CHECK:  release_value_addr [[BUF]] : $*Vector
```

Comment:
This should be changed to `destroy_addr`, not be deleted.

Reply:
Ah yes, thanks for the catch! I'll correct this in my next PR.